selfdrive/ui/qt/widgets/controls.h

```diff
@@ -42,7 +42,7 @@ class LabelControl : public AbstractControl {
   Q_OBJECT
 
 public:
-  LabelControl(const QString &title, const QString &text, const QString &desc = "", QWidget *parent = nullptr) : AbstractControl(title, desc, "", parent) {
+  LabelControl(const QString &title, const QString &text, const QString &desc, QWidget *parent = nullptr) : AbstractControl(title, desc, "", parent) {
```

Comment:
why change the default?

Reply:
This is indeed unnecessary, I changed it back